Makefile-Standalone

```diff
@@ -192,6 +192,14 @@ ifeq ($(ENABLE_CHIPOBLE_TEST),1)
 configure_OPTIONS               += --enable-chipoble-test=yes
 endif
 
+ifeq ($(HOSTOS),darwin)
+configure_OPTIONS               += --with-device-layer=darwin
```

Comment:
I think this is fine; however, I'd like to propose auto-detecting this in configure.ac by `--with-device-layer` supporting and defaulting to an `auto` value.

Reply:
This will preclude having this logic in both `Makefile-Standalone` and in `standalone-chip.mk`.

Reply:
Happy to help with this if you'd like.

Reply:
Makes sense. I have updated the patch.